permissions/plugin-jx-pipelines.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "jx-pipelines"
+paths:
+- "org/jenkins-ci/plugins/jx-pipelines"
+developers:
+- "jstrachan"
+- "jenkins-x-bot"
```

Comment:
This looks wrong FWICT: this user does not seem to exist on Jenkins infrastructure: https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=jenkins-x-bot

Reply:
@batmat whoops I'd used the githubuser name rather than the jenkins user name - all fixed thanks!